config/enabled.yml

```diff
@@ -590,6 +590,10 @@ Lint/NonLocalExitFromIterator:
   Description: 'Do not use return in iterator to cause non-local exit.'
   Enabled: true
 
+Lint/OrderedMagicComments:
+  Description: 'Checks coding magic comment should be first line, or next of shebang.'
```

Comment:
I'd suggest rewording this a bit. How about "Checks the proper ordering of magic comments and whether a magic comment is not placed before a shebang." or something along those lines.